sdk/typespec/ci.yml

```diff
@@ -14,6 +14,7 @@ extends:
   template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
     ServiceDirectory: typespec
+    RunLiveTests: false
```

Comment:
Can't we leave these as default and just let whether any `test-resources.*` files exist dictate whether to run them or not? Adding that + this variable duplicates effort.